README.md

```diff
@@ -1,5 +1,66 @@
 # Shoe Store
 
+## My work
+
+#### Installation
+
+```js
+$ npm install
+
+//web socket need to be started first
+$ websocketd --port=8080 ruby inventory.rb
+
+$npm start
+```
+
+This application is made with 
+- React and Redux.
+- A simple use of React-router
+- Styled-components library for the styling.
```

Comment:
### Why redux  : 
I think redux is a good state manager to use with React. It makes it easier to control the evolution of the state and to handle the re-rendering of the React components. 
Also, it's a good fit with the web-sockets, beacause it's easy to handle the response and to dispatch it to the reducer.

### Why styled-components
I used styled-components because I like it ! :smiley: It makes it easy to use CSS in JS and brings all of that power to the styling. You still write your CSS as if you are in a CSS file but you can add all the javascript logic in it easily. I believe it's the classic evolution of the styling thechnologies (react oriented). 
- It embraces react in the best way
- It makes the Html code more readable (removing all the classname, naming directly the div)
- Keeps CSS structure (with the template literals) and there is SASS support.
- You can easily extend components to style them.
- I didn't use the theming in this app but it can be easily controlled with styled-components.